main/src/library/List.flix

```diff
@@ -1394,4 +1394,18 @@ mod List {
         toArray(rc, l) !> Array.shuffle(rnd) |> Array.toList
     }
 
+    ///
+    /// Returns the frequency for each element in list `l`
+    ///
+    pub def frequency(l: List[t]): Map[t, Int32] with Order[t] =
+        def freq(ll, m) = match ll {
+            case List.Nil   => m
+            case (x::xs)    =>
```

Comment:
This should be `x :: xs`, no need for parens

Reply:
I'm going to change the code.